examples/tutorials/vectors.py

```diff
@@ -0,0 +1,459 @@
+
+# create a plot with coast, Mercator projection (M) over the continental US
+fig = pygmt.Figure()
+fig.coast(
+    region=[-127, -64, 24, 53],
+    projection="M15c",
+    frame="ag",
+    borders=1,
+    area_thresh=4000,
+    shorelines="0.25p,black",
+    land="grey",
+    water="lightblue",
+)
+
+style = "v0.6c+bc+ea+a30"
+fig.plot(
+    x=-110,
+    y=40,
+    style=style,
+    pen="1p",
+    color="red3",
+    direction=[[-25], [3]],
+)
+
+# vector specifications structured as: [x_start, y_start, direction_degrees, length]
+vector_2 = [-82, 40.5, 138, 3]
+vector_3 = [-71.2, 45, -115.7, 6]
+# Create a list of lists that include each vector information
+vectors = np.array([vector_2] + [vector_3])
+
+fig.plot(
+    data=vectors,
+    style=style,
+    pen="1p",
+    color="yellow",
+)
+
+fig.show()
+
+########################################################################################
+# Another example of plotting cartesian vectors over a coast plot. This time
+# a Transverse Mercator projection is used. Additionally, ``numpy.linespace``
+# is used to create 5 vectors with equal stops.
+
+# create a plot with coast, Mercator projection (M) over the continental US
+fig = pygmt.Figure()
+fig.coast(
+    region=[20, 50, 30, 45],
+    projection="T35/12c",
+    frame=True,
+    borders=1,
+    area_thresh=4000,
+    shorelines="0.25p,black",
+    land="lightbrown",
+    water="lightblue",
+)
+
+x = np.linspace(36, 42, 5)  # x values = [36.  37.5 39.  40.5 42. ]
+y = np.linspace(39, 39, 5)  # y values = [39. 39. 39. 39.]
+direction = np.linspace(-90, -90, 5)  # direction values = [-90. -90. -90. -90.]
+length = np.linspace(1.5, 1.5, 5)  # length values = [1.5 1.5 1.5 1.5]
+
+fig.plot(
+    x=x,
+    y=y,
+    style="v0.4c+ea",
+    pen="0.6p",
+    color="red3",
+    direction=[direction, length],
+)
+
+
+fig.show()
+
+########################################################################################
+# Plot Circular Vectors
+# ---------------------
+#
+# When plotting circular vectors, there are 5 values that should be included in
+# the list that is passed through ``np.array()`` in order to create a valid plot.
+# The first two values in ``circular_vector_1`` represent the origin of the
+# circle that will be plotted. The next value is the radius which is represented
+# on the plot in centimeters. Finally, the last two values represent the degree
+# at which the plot will start and stop. In this example, the result shown is the
+# left half of a circle as the plot starts at 90 degrees and goes until 270.
+# It is important to note that when plotting circular vectors, the style value
+# should begin with an ``m``.
+
+# vector specifications structured as: [x_start, y_start, radius, degree_start, degree_stop]
+circular_vector_1 = [0, 0, 5, 90, 270]
+
+data = np.array([circular_vector_1])
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[-10, 10, -10, 10],
+    projection="X20c",
```

Comment:
Avoid too large images:
```suggestion
    projection="X10c",
```

NOTE: the radius in `circular_vector_1 = [0, 0, 5, 90, 270]` may also need to be changed.